packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -81,10 +83,19 @@ export function PanelChrome({
   statusMessage,
   statusMessageOnClick,
   leftItems,
+  onCancelQuery,
 }: PanelChromeProps) {
   const theme = useTheme2();
   const styles = useStyles2(getStyles);
-  const hasHeader = !hoverHeader;
+
+  const pointerQuery = useMemo(() => '(pointer: coarse)', []); // memoize the media query string
```

Comment:
????

Reply:
ups a mistake, I was trying to do something else and forgot to remove this :)